### PR TITLE
Frontend stuff

### PR DESCRIPTION
**Pull request by Levis92**

### CODE REVIEW

File: react-app/src/components/SearchBar.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ class SearchBar extends Component {
     this.previous_input = "";
   }
 
-  tokenize_query(query) {
+  jsonify_query(query) {
```
Comment (lindskogen) [inline react-app/src/components/SearchBar.js]: Inte för att vara sån, men...? 😛

File: react-app/src/components/SearchBar.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ class SearchBar extends Component {
 
   request() {
     request
-      .get(baseurl + this.jsonify_query(this.previous_input))
+      .get(baseurl + this.tokenize_query(this.previous_input))
```
Comment (lindskogen) [inline react-app/src/components/SearchBar.js]: ❤️

File: react-app/src/components/SearchBar.js [hunk 3]
```diff
@@ -42,25 +42,24 @@ class SearchBar extends Component {
     }
   }
 
+  search = value => {
+    this.previous_input = value;
```
Comment (lindskogen) [inline react-app/src/components/SearchBar.js]: What is the purpose of the `previous_input` property?
Why not store the input's value in state?

```js
this.setState({ value });

// then in request():

.get(baseurl + this.tokenize_query(this.state.value))

File: react-app/src/components/SearchBar.js [hunk 4]
```diff
@@ -42,25 +42,24 @@ class SearchBar extends Component {
...
           placeholder="nick 0, nick 1, ..."
-          onChange={this.search.bind(this)}
+          onChange={e => this.search(e.target.value)}
         />
       </div>
     );
   }
-
-  search() {
-    var raw_input = document.getElementById("search-input").value;
```
Comment (lindskogen) [inline react-app/src/components/SearchBar.js:60]: Good! No direct DOM-manipulation ;)

**Review by HerrNaN — APPROVED** [review]

**Pull request merged**